### ISSUE TITLE
Changing appearance of gene track

### ISSUE THREAD

**Issue by alhafidzhamdan** [GreenleafLab/ArchR#1020]
Hi there,
I just have a simple question- how would I change the appearance of the gene track? Atm the genes are not easily visible using the default settings.
```

p <- plotBrowserTrack(
    ArchRProj = E34_filtered, 
    groupBy = "LSI_clusters", 
    log2Norm = T,
    geneSymbol = markerGenes, 
    upstream = 50000,
    downstream = 50000,
    borderWidth = 0.4,
    facetbaseSize =9,
)
grid::grid.newpage()
grid::grid.draw(p$DDIT3)

```

![image](https://user-images.githubusercontent.com/53702866/132209758-2bbff886-50cf-4ec0-b576-2d8dd7ee73ca.png)

Thanks in advance, appreciate your support
A

**Comment by rcorces** [comment]
Duplicate of https://github.com/GreenleafLab/ArchR/issues/977
Follow up there if you would like further assistance. This is a problem with some dependency or some system configuration. Since it isnt a problem with ArchR per se and I cannot replicate the problem, its hard for me to figure out the cause. You're welcome to see if you can figure it out. It could be a ggplot2 version issue?

**Comment by cristanchoa** [comment]
I was having the same issue. I just wanted to comment that this fix worked well when I updated to 1.0.2. Thank you!
